### PR TITLE
HRTIM

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

File: peripherals/hrtim/hrtim.yaml [hunk 1]
```diff
@@ -409,25 +222,51 @@ HRTIM_Master:
...
-      Continuous: [1, "The timer operates in continuous (free-running) mode and rolls over to zero when it reaches the MPER value"]
+      SingleShot:
+        [
+          0,
+          "The timer operates in single-shot mode and stops when it reaches the MPER value",
+        ]
+      Continuous:
+        [
+          1,
+          "The timer operates in continuous (free-running) mode and rolls over to zero when it reaches the MPER value",
```
Comment (usbalbin) [inline peripherals/hrtim/hrtim.yaml:244]: `MPER` should be `PERxR` here, right?

**Comment by usbalbin** [comment]
I am not too comfortable with svd2rust but as far as I can tell it looks good!

The resulting changes to my hrtim branch [here](https://github.com/usbalbin/stm32g4xx-hal/compare/hrtim6...usbalbin:stm32g4xx-hal:hrtim_with_stm32-rs%231021?expand=1) in case it is useful

**Comment by burrbull** [comment]
@usbalbin I cannot approve myself. Could you open a new PR?

**Review by usbalbin — APPROVED** [review]

**Comment by usbalbin** [comment]
Does that help?

**Comment by burrbull** [comment]
No

**Comment by usbalbin** [comment]
I pressed the "Review changes" and "Approve"

image

Any more buttons to press? :)

**Comment by burrbull** [comment]
You are not a maintainer of this repository. So your approve has no affect on possibility to merge.

**Comment by usbalbin** [comment]
Oh. I dont think I can open a PR for this branch while this PR is open

**Comment by usbalbin** [comment]
Done #1022

**Pull request closed without merging**